setup.py

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env python
+from setuptools import setup, find_packages
+
+DISTNAME = 'mne_g3d'
```

Comment:
why "g"? I was thinking "j3d" for "Jupyter", but if you like "g" for "Google" it's fine by me (will be changed eventually anyway)

Reply:
Oh, that's what 'j' supposed to mean, yeah I thought to put 'g' from Google or GSOC, I have created `mne_g3d` folder and inside it `mne_g3d/__init__.py`. I was planning to add `viz.py` later while working on the #1, will it ok?

Reply:
Yep